packages/nodes-base/nodes/Todoist/Todoist.node.ts

```diff
@@ -153,11 +154,10 @@ export class Todoist implements INodeType {
 					},
 				],
 				default: 'create',
-				description: 'The operation to perform.',
 			},
 			{
 				displayName: 'Project',
-				name: 'project',
+				name: 'projectId',
```

Comment:
Renaming a parameter is a breaking change. So should be reversed.

Reply:
Is there a process for long-term changes like this one? The node is a bit messy, with some more recent additions using the `Id` suffix while others don't.

Reply:
At some point, would it then make sense to create a totally new version of the node.

Reply:
Ah, that's a great idea. Or versioned nodes. 🤔 Combined with a way to keep track of all of these “wishes” (e.g., a top-level comment), it could be a powerful way to keep polishing the platform without necessarily being locked-in by past decisions.